src/PrizePool.sol

```diff
@@ -484,14 +499,29 @@ contract PrizePool is Manageable, Multicall {
             claimCount++;
         }
         claimRecords[_winner] = ClaimRecord({drawId: lastCompletedDrawId, claimedTiers: uint8(BitLib.flipBit(claimRecord.claimedTiers, _tier))});
+        UD60x18 deltaExchangeRate = _computeTierDeltaExchangeRate(prizeSize, _tier);
+        // console2.log("deltaExchangeRate", deltaExchangeRate.unwrap());
+        // console2.log("_tierExchangeRates[_tier]", _tierExchangeRates[_tier].unwrap());
+        UD60x18 test = ud(_tierExchangeRates[_tier].unwrap() + deltaExchangeRate.unwrap());
+        _tierExchangeRates[_tier] = _tierExchangeRates[_tier].add(test);
+        // console2.log("GOT HERE????");
         prizeToken.transfer(_to, payout);
         if (_fee > 0) {
             claimerRewards[_feeRecipient] += _fee;
         }
         emit ClaimedPrize(lastCompletedDrawId, _vault, _winner, _tier, uint152(payout), _to, _fee, _feeRecipient);
+        // console2.log("ALLL DONE");
         return prizeSize;
     }
 
+    function _computeTierDeltaExchangeRate(uint256 _prizeSize, uint8 _tier) internal view returns (UD60x18) {
+        if (_tierExchangeRates[_tier].gte(prizeTokenPerShare)) {
```

Comment:
Store `prizeTokenPerShare` and `_tierExchangeRates[_tier]` in variables.

Reply:
We also have the same check in `_getLiquidity`:
https://github.com/pooltogether/v5-prize-pool/blob/01a28037121ed67496fc547567e45ebeb91b1ab2/src/PrizePool.sol#L718

We can compute `_getLiquidity(_tier, tierShares)` and we return it if it returns 0;